docs/framework/wcf/extending/how-to-import-custom-wsdl.md

```diff
@@ -67,8 +67,9 @@ This topic describes how to import custom WSDL. To handle the custom WSDL, you m
   
 3. Create a new <xref:System.ServiceModel.Description.WsdlImporter> instance (passing in the <xref:System.ServiceModel.Description.MetadataSet> instance that contains the WSDL documents that you want to import), and call <xref:System.ServiceModel.Description.WsdlImporter.ImportAllContracts%2A>:  
   
-    ```  
-    WsdlImporter importer = new WsdlImporter(metaDocs);          System.Collections.ObjectModel.Collection<ContractDescription> contracts  = importer.ImportAllContracts();  
+    ```csharp
+    WsdlImporter importer = new WsdlImporter(metaDocs);
+	System.Collections.ObjectModel.Collection<ContractDescription> contracts = importer.ImportAllContracts();  
```

Comment:
The tab character is causing extra indentation on the second line.  I'd suggest the following:

```suggestion
    System.Collections.ObjectModel.Collection<ContractDescription> contracts = importer.ImportAllContracts();  
```